pkg/util/query/query.go

```diff
@@ -0,0 +1,24 @@
+package query
```

Comment:
only used within `kubeobjects`, even its name is `KubeQuery`, so it should be under `kubeobjects`

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/2336/commits/d7ce7d507fe13a29517cf88b4a4f5234f12425e0